OneCommunityGlobal/HighestGoodNetworkApp
# Natalia make reports look good on phone

### Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/103126226/842e2aee-dbda-4dcd-a89b-797384713aef)

### Main changes explained
To fix responsiveness on the reports page, and every subsequent page. 

### How to test
1. check into current branch
2. `npm install` 
3. `npm run start:local` to run this PR locally
5. log in as Admin/Owner, go to Reports → Reports
6. verify if all the pages are responsive. 
- Click the Projects/People/Teams buttons, and click at least one item from each table.

### Note
- Please test for Mobile 375px minimum screen size. 
- The graphs are with overflow: scroll for mobiles, for now, this was the best solution to not lose any data.

### Screenshots of changes:
Before:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/103126226/e04daf22-87d1-4a57-b5e7-5b8c94d1d58b)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/103126226/dce6bd50-5ee5-477a-a9f4-2d872c9d10b7)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/103126226/786bad51-0cb7-45d6-b4c5-d88f4225f2d3)



After:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/103126226/754a6bed-8348-46e1-a15d-29c0f22c4319)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/103126226/db353d85-9f2c-41cd-b5f0-43f8ad8ee451)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/103126226/d39225ad-f535-42fc-a270-d0b1d7d9d9e2)
